twisted_s3/client.py

```diff
@@ -53,10 +53,17 @@ def _make_request(
 
         hashed_payload = auth.compute_hashed_payload(payload)
 
-        host = "{bucket}.s3-{region}.amazonaws.com".format(
-            bucket=bucket,
-            region=region
-        )
+        host = ""
```

Comment:
So now that I understand the issue I'd prefer you do this:

```python
host_template = US_EAST_TEMPLATE if region == "us-east-1" else HOST_TEMPLATE
host = host_template.format(bucket=bucket, region=region)
```

where you define the templates as global constants. Since this uses named arguments for string formatting it doesn't care that the `US_EAST_TEMPLATE` takes a kwarg it doesn't use.